google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/facet/flexible/AppEngineFlexibleSupportProvider.java

```diff
@@ -99,6 +101,9 @@ public boolean isSupportAlreadyAdded(@NotNull Module module,
   public static void addSupport(@NotNull AppEngineFlexibleFacet facet,
       @NotNull ModifiableRootModel rootModel,
       boolean generateConfigFiles) {
+    UsageTrackerProvider.getInstance()
+        .trackEvent(GctTracking.APP_ENGINE_ADD_SUPPORT)
+        .addMetadata("environment", "flex");
```

Comment:
this isn't completely consistent with the standard equivalent ping in this place